codetools/contexts/tests/data_context_test_case.py

```diff
@@ -1,15 +1,33 @@
 
 # Standard Library Imports
 from io import BytesIO
+import os
 import sys
 
+from importlib_resources import files
 import nose
 
 # Local library imports
 from codetools.contexts.data_context import DataContext
 from codetools.contexts.tests.abstract_context_test_case import AbstractContextTestCase
 
 
+def create_data_context_pickle():
```

Comment:
let's make it slightly more obvious that this is private

```suggestion
def _create_data_context_pickle():
```

Reply:
I initially thought we should just remove the function altogether but it's good to know how the pickle was created.